<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APhotonTunnelWindow</class>
 <widget class="QMainWindow" name="APhotonTunnelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photon tunnels</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="twMain">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Models</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Connection</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>140</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Sort by:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbSortByFrom">
             <property name="text">
              <string>From</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbSortByTo">
             <property name="text">
              <string>To</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>139</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Already defined connections:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tabwConnections">
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>From</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>To</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Model</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Settings</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>From:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sbFrom">
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="maximum">
                <number>9999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>To:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sbTo">
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="maximum">
                <number>9999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Model:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="sbModel">
               <property name="buttonSymbols">
                <enum>QAbstractSpinBox::NoButtons</enum>
               </property>
               <property name="maximum">
                <number>9999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Extra settings:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leSettings"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pbAddModify">
               <property name="text">
                <string>Add / Modify</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbRemove">
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Show connection on geometry window</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Info for the selected connection:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
